spec/search/parens.spec.ts

```diff
@@ -1,24 +1,26 @@
-const handleParens = require('../../src/services/search/services/handle_parens');
+import handleParens from "../../src/services/search/services/handle_parens";
```

Comment:
Then should all file imports have the extension?

Reply:
@JYC333 , that would be correct, yes. Initially the `tsconfig` was improperly configured, causing these issues to not show up as errors.

They are handled in: https://github.com/TriliumNext/Notes/pull/268